automation_Test/actions.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#!/usr/bin/env python
+
+import time
+import os
+from time import sleep
+import constants as const
+from appium import webdriver
+
+def getWebDriver():
+    desired_caps = {}
+    desired_caps['appium-version'] = const.K_APPIUM_VER
+    desired_caps['platformName'] = const.K_APP_PLATFORM_NAME
+    desired_caps['platformVersion'] = const.K_APP_PLATFORM_VER
+    desired_caps['deviceName'] = const.K_DEVICE_NAME
+    return webdriver.Remote(const.K_WD_REMOTE_URL, desired_caps)
+
+def doLoginWith(driver,server,user,password, ssl):
+    user_field = driver.find_elements_by_class_name('UIATextField')[1]
+    user_field.clear()
+    user_field.set_value(user)
+
+    pass_field = driver.find_elements_by_class_name('UIASecureTextField')[0]
+    pass_field.clear()
+    pass_field.set_value(password)
+
+    url_field = driver.find_elements_by_class_name('UIATextField')[0]
+    url_field.clear()
+    url_field.set_value(server)
```

Comment:
I will add here some code to wait up until Connection is stablished has been shown. in this way, it would not try to click on a button that maybe is not enable yet.